build/utils/const.js

```diff
@@ -6,14 +6,13 @@ import path from 'path';
 export const ROOT = path.resolve(path.dirname(path.join(__dirname, '..')));
 
 export const BUILD_SYSTEM_DIR = path.join(ROOT, 'build');
-
 export const SRC_DIR = path.join(ROOT, 'app');
 export const LIB_DIR = path.join(ROOT, 'lib');
-export const APP_SHARED_DIR = path.join(SRC_DIR, 'shared');
+export const PACKAGED_DIST_DIR = path.join(ROOT, 'dist');
 
+export const APP_SHARED_DIR = path.join(SRC_DIR, 'shared');
 export const BUILD_CONFIG_PATH = path.join(SRC_DIR, 'build-config.json');
-
-export const PACKAGED_DIST_DIR = path.join(ROOT, 'dist');
+export const BROWSER_FRONTENDS_PATH = path.join(BUILD_SYSTEM_DIR, 'config', 'frontends.json');
```

Comment:
I'd like having this list in a separate file. That way whenever a new frontend is added, all that is needed is editing that array and creating a new webpack config file. I personally don't see any relevant difference between having a js file exporting an array, or a json file that's being loaded. Feel free to let me know if you feel strongly about this.